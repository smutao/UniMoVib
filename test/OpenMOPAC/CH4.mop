 AM1 FORCE DFORCE

CH4. The geometry has been optimized by the keywords AM1 GRADIENTS
C   0.0000 0    0.0000 0    0.0000 0    0 0 0
H   1.1116 0    0.0000 0    0.0000 0    1 0 0
H   1.1116 0  109.4712 0    0.0000 0    1 2 0
H   1.1116 0  109.4712 0  120.0000 0    1 2 3
H   1.1116 0  109.4712 0 -120.0000 0    1 2 3

