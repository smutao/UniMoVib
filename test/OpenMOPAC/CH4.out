 *******************************************************************************
 ** Site#: 21400        For non-commercial use only     Version 14.311W 32BITS**
 *******************************************************************************
 ** Cite this work as:                                                        **
 **                                                                           **
 ** (a) MOPAC2012, James J. P. Stewart, Stewart Computational Chemistry,      **
 **  Version 14.311W web: HTTP://OpenMOPAC.net         Days remaining: 360    **
 **                                                                           **
 **                                 AND                                       **
 **                                                                           **
 ** (b) Maia, J.D.C., et al., J. Chem. Theory Comput. 2012, 8, 3072-3081.     **
 **                DOI: 10.1021/ct3004645.                                    **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2012                                  **
 **                                                                           **
 *******************************************************************************

                            AM1 CALCULATION RESULTS


 *******************************************************************************
 *  CALCULATION DONE:                                Wed Nov 12 07:19:05 2014  *
 *  AM1        - The AM1 Hamiltonian to be used
 *  FORCE      - FORCE CALCULATION SPECIFIED
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *  DFORCE     - PRINT FORCE MATRIX OVER CARTESIAN COORDINATES
 *******************************************************************************
 AM1 FORCE DFORCE

 CH4. The geometry has been optimized by the keywords AM1 GRADIENTS
  ATOM    CHEMICAL      BOND LENGTH      BOND ANGLE     TWIST ANGLE 
 NUMBER    SYMBOL       (ANGSTROMS)      (DEGREES)       (DEGREES) 
   (I)                     NA:I           NB:NA:I       NC:NB:NA:I       NA    NB    NC 
     1       C          0.00000000       0.0000000       0.0000000        0     0     0
     2       H          1.11160000       0.0000000       0.0000000        1     0     0
     3       H          1.11160000     109.4712000       0.0000000        1     2     0
     4       H          1.11160000     109.4712000     120.0000000        1     2     3
     5       H          1.11160000     109.4712000    -120.0000000        1     2     3
 


          CARTESIAN COORDINATES 

    NO.       ATOM         X         Y         Z

     1         C        0.0000    0.0000    0.0000
     2         H        1.1116    0.0000    0.0000
     3         H       -0.3705    1.0480    0.0000
     4         H       -0.3705   -0.5240   -0.9076
     5         H       -0.3705   -0.5240    0.9076

     ===================== shared-memory parallel CPU Section ======================
     **                                                                           **
     **  MOPAC calculations will run in shared-memory parallel CPUs               **
     **                                                                           **
     **  Number of Threads =  2                                                   **
     **                                                                           **
     **                                                                           **
     **  For information on this work, see:                                       **
     **                                                                           **
     **  Maia, J.D.C., et al., J. Chem. Theory Comput. 2012, 8, 3072-3081.        **
     **        DOI: 10.1021/ct3004645.                                            **
     **                                                                           **
     ===================== shared-memory parallel CPU Section ======================
      


  H: (AM1): M.J.S. DEWAR ET AL, J. AM. CHEM. SOC. 107 3902-3909 (1985)
  C: (AM1): M.J.S. DEWAR ET AL, J. AM. CHEM. SOC. 107 3902-3909 (1985)

           Empirical Formula: C H4  =     5 atoms



      MOLECULAR POINT GROUP   :   Td  


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =    4



          HEAT OF FORMATION =      -8.789787 KCALS/MOLE


          INTERNAL COORDINATE DERIVATIVES

   NUMBER  ATOM       BOND         ANGLE      DIHEDRAL

     1       C     -0.000158     0.000001     0.000000
     2       H     -0.025107     0.000000     0.000000
     3       H      0.008421    -0.023551     0.000371
     4       H      0.008422     0.012096     0.020210
     5       H      0.008422     0.011454    -0.020581


          GRADIENT NORM =   0.05007


          TIME FOR SCF CALCULATION =    0.04

          TIME FOR DERIVATIVES     =    0.01

          MOLECULAR WEIGHT =   16.04



          ROTATIONAL CONSTANTS IN CM(-1)

          A =    5.07590349   B =    5.07590349   C =    5.07590155



           PRINCIPAL MOMENTS OF INERTIA IN UNITS OF 10**(-40)*GRAM-CM**2

          A =        5.5148   B =        5.5148   C =        5.5148


          ORIENTATION OF MOLECULE IN FORCE CALCULATION

    NO.       ATOM         X         Y         Z

     1          C      -0.0000   -0.0000    0.0000
     2          H       1.1116   -0.0000    0.0000
     3          H      -0.3705    1.0480    0.0000
     4          H      -0.3705   -0.5240   -0.9076
     5          H      -0.3705   -0.5240    0.9076


    FIRST DERIVATIVES WILL BE USED IN THE CALCULATION OF SECOND DERIVATIVES

          ESTIMATED TIME TO COMPLETE CALCULATION =        1.52 SECONDS

    FOR POINT-GROUP Td   THERE ARE  24 UNIQUE SYMMETRY FUNCTIONS.

 STEP:   1 TIME =     0.00 SECS, INTEGRAL =      0.00 TIME LEFT: 172799.95
 STEP:   2 TIME =     0.00 SECS, INTEGRAL =      0.01 TIME LEFT: 172799.94
 STEP:   3 TIME =     0.00 SECS, INTEGRAL =      0.01 TIME LEFT: 172799.94
 STEP:   4 TIME =     0.00 SECS, INTEGRAL =      0.01 TIME LEFT: 172799.94
 STEP:   5 TIME =     0.00 SECS, INTEGRAL =      0.01 TIME LEFT: 172799.94
 STEP:   6 TIME =     0.00 SECS, INTEGRAL =      0.01 TIME LEFT: 172799.94
 STEP:   7                       INTEGRAL =      0.01 TIME LEFT: 172799.94
 STEP:   8                       INTEGRAL =      0.01 TIME LEFT: 172799.94
 STEP:   9                       INTEGRAL =      0.01 TIME LEFT: 172799.94
 STEP:  10                       INTEGRAL =      0.01 TIME LEFT: 172799.94
 STEP:  11                       INTEGRAL =      0.01 TIME LEFT: 172799.94
 STEP:  12                       INTEGRAL =      0.01 TIME LEFT: 172799.94
 STEP:  13                       INTEGRAL =      0.01 TIME LEFT: 172799.94
 STEP:  14                       INTEGRAL =      0.01 TIME LEFT: 172799.94
 STEP:  15                       INTEGRAL =      0.01 TIME LEFT: 172799.94


           FULL FORCE MATRIX, INVOKED BY "DFORCE"


                     1          2          3          4          5          6
 ------------------------------------------------------------------------------
          1   7.888914
          2  -0.000005   7.889330
          3   0.000019   0.000029   7.889705
          4  -4.607484  -0.000029  -0.000940   5.451682
          5   0.000018  -0.654942  -0.000025  -0.000045   0.657353
          6   0.000018   0.000120  -0.654868   0.000244  -0.000216   0.656531
          7  -1.093803   1.242248  -0.000014  -0.281300   0.464335   0.000306
          8   1.242072  -4.168356  -0.000039   0.142906  -0.066666   0.000070
          9  -0.000242   0.000036  -0.654903   0.000571   0.000002   0.065349
         10  -1.093803  -0.621091  -1.075827  -0.281447  -0.232171  -0.402832
         11  -0.621245  -1.533063  -1.521431  -0.071439   0.032193  -0.056999
         12  -1.075545  -1.521345  -3.289976  -0.123302  -0.057102  -0.033544
         13  -1.093803  -0.621091   1.075807  -0.281473  -0.232159   0.402505
         14  -0.620826  -1.533124   1.521430  -0.071415   0.032217   0.057120
         15   1.075787   1.521309  -3.289907   0.124105   0.057131  -0.033520


                     7          8          9         10         11         12
 ------------------------------------------------------------------------------
          7   1.190087
          8  -1.506750   4.918948
          9  -0.000285   0.000159   0.656531
         10   0.092759   0.060863   0.080539   1.189461
         11  -0.099869  -0.342013  -0.398793   0.753401   1.722899
         12  -0.012735  -0.135284  -0.033544   1.305286   1.845375   3.853206
         13   0.092766   0.060899  -0.080316   0.093136   0.039232  -0.092710
         14  -0.099845  -0.342022   0.398525   0.039237   0.120463  -0.131311
         15   0.012495   0.136051  -0.033520   0.092881   0.131179  -0.496375


                    13         14         15
 ------------------------------------------
         13   1.189460
         14   0.753710   1.722489
         15  -1.305106  -1.845138   3.853617

(To convert to Hartree/Bohr^2, multiply by (10^(-21) x N x a0^2)/(4.184 x 627.51) =  0.0642)


           FORCE MATRIX IN MILLIDYNES/ANGSTROM


                  C  1       H  2       H  3       H  4       H  5
 ------------------------------------------------------------------
     C    1  13.664697
     H    2   4.699651   5.530279
     H    3   4.699619   0.569099   5.530279
     H    4   4.699550   0.569532   0.569532   5.530279
     H    5   4.699505   0.569495   0.569495   0.569495   5.530279


          HEAT OF FORMATION =      -8.789787 KCALS/MOLE

          ZERO POINT ENERGY      27.855 KCAL/MOL


           NORMAL COORDINATE ANALYSIS (Total motion = 1 Angstrom)


   Root No.       1         2         3         4         5         6         7         8

                 1 A       2 A       3 A       4 A       5 A       6 A       7 A       8 A   

                1379.6    1379.6    1379.6    1411.3    1411.3    3102.9    3102.9    3102.9
  
           1    0.0000   -0.0682   -0.0001   -0.0000    0.0000   -0.0476    0.0000    0.0000
           2   -0.0643   -0.0000   -0.0000    0.0000   -0.0000   -0.0000   -0.0492   -0.0000
           3    0.0000    0.0001   -0.0633   -0.0000    0.0000   -0.0000    0.0000   -0.0547
           4    0.0000   -0.0288   -0.0001   -0.0000    0.0000    0.4738   -0.0000   -0.0000
           5    0.3009    0.0000    0.0000    0.2500    0.0000   -0.0000   -0.0249   -0.0000
           6   -0.0000   -0.0006    0.2961   -0.0000    0.2500   -0.0007    0.0000   -0.0277
           7    0.1031    0.2805    0.0006    0.2357    0.0000    0.0312   -0.1618   -0.0001
           8    0.0093    0.1093    0.0002    0.0833    0.0000   -0.1564    0.4327    0.0004
           9   -0.0004   -0.0007    0.2961   -0.0003   -0.2500    0.0002    0.0001   -0.0277
          10   -0.0515    0.2807   -0.0873   -0.1178   -0.2041    0.0307    0.0808    0.1558
          11    0.2276   -0.0544   -0.1243   -0.1670    0.1443    0.0782    0.0893    0.2203
          12   -0.1260   -0.0949    0.0807    0.1445    0.0000    0.1357    0.1979    0.3540
          13   -0.0516    0.2802    0.0884   -0.1179    0.2041    0.0317    0.0810   -0.1557
          14    0.2283   -0.0549    0.1241   -0.1664   -0.1444    0.0782    0.0896   -0.2202
          15    0.1264    0.0945    0.0811   -0.1442   -0.0000   -0.1353   -0.1984    0.3536


   Root No.       9

                 9 A   

                3214.8
  
           1    0.0000
           2    0.0000
           3    0.0000
           4    0.2500
           5    0.0000
           6   -0.0004
           7   -0.0833
           8    0.2357
           9    0.0001
          10   -0.0836
          11   -0.1179
          12   -0.2040
          13   -0.0830
          14   -0.1179
          15    0.2042


           MASS-WEIGHTED COORDINATE ANALYSIS (NORMAL COORDINATES)


   Root No.       1         2         3         4         5         6         7         8

                 1 A       2 A       3 A       4 A       5 A       6 A       7 A       8 A   

                1379.6    1379.6    1379.6    1411.3    1411.3    3102.9    3102.9    3102.9
  
           1    0.0000   -0.4110   -0.0009   -0.0000    0.0000   -0.2871    0.0000    0.0000
           2   -0.4110   -0.0000   -0.0000    0.0000    0.0000   -0.0000   -0.2871   -0.0002
           3    0.0000    0.0009   -0.4110   -0.0000   -0.0000   -0.0000    0.0002   -0.2871
           4    0.0000   -0.0502   -0.0001   -0.0000    0.0000    0.8275   -0.0000   -0.0001
           5    0.5572    0.0000    0.0000    0.5000    0.0001   -0.0000   -0.0421   -0.0000
           6   -0.0000   -0.0011    0.5571   -0.0001    0.5000   -0.0012    0.0000   -0.0421
           7    0.1909    0.4897    0.0011    0.4714    0.0001    0.0545   -0.2733   -0.0002
           8    0.0173    0.1909    0.0004    0.1667    0.0000   -0.2732    0.7308    0.0006
           9   -0.0008   -0.0012    0.5571   -0.0006   -0.5000    0.0004    0.0001   -0.0421
          10   -0.0954    0.4901   -0.1643   -0.2356   -0.4083    0.0536    0.1364    0.2368
          11    0.4215   -0.0949   -0.2340   -0.3340    0.2886    0.1367    0.1509    0.3348
          12   -0.2334   -0.1657    0.1519    0.2890    0.0000    0.2371    0.3342    0.5378
          13   -0.0955    0.4892    0.1663   -0.2358    0.4082    0.0554    0.1368   -0.2366
          14    0.4228   -0.0959    0.2336   -0.3327   -0.2887    0.1366    0.1513   -0.3346
          15    0.2341    0.1650    0.1526   -0.2884   -0.0000   -0.2362   -0.3351    0.5373


   Root No.       9

                 9 A   

                3214.8
  
           1    0.0000
           2    0.0000
           3    0.0000
           4    0.5000
           5    0.0000
           6   -0.0007
           7   -0.1667
           8    0.4714
           9    0.0002
          10   -0.1672
          11   -0.2357
          12   -0.4080
          13   -0.1661
          14   -0.2357
          15    0.4085


          DESCRIPTION OF VIBRATIONS


 VIBRATION    1    1A       ATOM PAIR        ENERGY CONTRIBUTION    RADIAL
 FREQ.      1379.62        C  1 --  H  2          +34.5% ( 59.7%)     0.0%
 T-DIPOLE    0.1695        C  1 --  H  5          +28.2% ( 54.0%)     3.8%
 TRAVEL      0.1415        C  1 --  H  4          +28.1% ( 53.9%)     3.8%
 RED. MASS   0.5592        C  1 --  H  3           +9.2% ( 30.8%)    52.6%
 EFF. MASS   2.4423

 VIBRATION    2    2A       ATOM PAIR        ENERGY CONTRIBUTION    RADIAL
 FREQ.      1379.62        C  1 --  H  4          +31.4% ( 56.9%)     1.7%
 T-DIPOLE    0.1695        C  1 --  H  3          +31.3% ( 56.9%)     1.7%
 TRAVEL      0.1415        C  1 --  H  5          +31.3% ( 56.8%)     1.7%
 RED. MASS   0.5733        C  1 --  H  2           +6.0% ( 25.0%)   100.0%
 EFF. MASS   2.4423

 VIBRATION    3    3A       ATOM PAIR        ENERGY CONTRIBUTION    RADIAL
 FREQ.      1379.62        C  1 --  H  3          +34.5% ( 59.7%)     0.0%
 T-DIPOLE    0.1695        C  1 --  H  2          +34.5% ( 59.7%)     0.0%
 TRAVEL      0.1415        C  1 --  H  5          +15.6% ( 40.1%)    21.6%
 RED. MASS   0.5592        C  1 --  H  4          +15.5% ( 40.0%)    21.8%
 EFF. MASS   2.4423

 VIBRATION    4    4A       ATOM PAIR        ENERGY CONTRIBUTION    RADIAL
 FREQ.      1411.33        C  1 --  H  4          +25.1% ( 37.3%)     0.0%
 T-DIPOLE    0.0000        C  1 --  H  3          +25.0% ( 37.3%)     0.0%
 TRAVEL      0.2177        C  1 --  H  2          +25.0% ( 37.3%)     0.0%
 RED. MASS   0.2520        C  1 --  H  5          +24.9% ( 37.2%)     0.0%
 EFF. MASS   1.0079

 VIBRATION    5    5A       ATOM PAIR        ENERGY CONTRIBUTION    RADIAL
 FREQ.      1411.33        C  1 --  H  2          +25.0% ( 37.3%)     0.0%
 T-DIPOLE    0.0000        C  1 --  H  3          +25.0% ( 37.3%)     0.0%
 TRAVEL      0.2177        C  1 --  H  4          +25.0% ( 37.3%)     0.0%
 RED. MASS   0.2520        C  1 --  H  5          +25.0% ( 37.3%)     0.0%
 EFF. MASS   1.0079

 VIBRATION    6    6A       ATOM PAIR        ENERGY CONTRIBUTION    RADIAL
 FREQ.      3102.91        C  1 --  H  2          +73.3% ( 87.9%)   100.0%
 T-DIPOLE    0.0320        C  1 --  H  4           +8.9% ( 30.6%)    72.3%
 TRAVEL      0.1215        C  1 --  H  5           +8.9% ( 30.6%)    71.9%
 RED. MASS   0.5723        C  1 --  H  3           +8.9% ( 30.6%)    72.1%
 EFF. MASS   1.4730

 VIBRATION    7    7A       ATOM PAIR        ENERGY CONTRIBUTION    RADIAL
 FREQ.      3102.91        C  1 --  H  3          +65.3% ( 83.0%)    99.4%
 T-DIPOLE    0.0320        C  1 --  H  5          +17.0% ( 42.3%)    85.6%
 TRAVEL      0.1215        C  1 --  H  4          +16.9% ( 42.2%)    85.5%
 RED. MASS   0.5026        C  1 --  H  2           +0.8% (  9.5%)     0.0%
 EFF. MASS   1.4730

 VIBRATION    8    8A       ATOM PAIR        ENERGY CONTRIBUTION    RADIAL
 FREQ.      3102.91        C  1 --  H  4          +49.2% ( 72.0%)    97.6%
 T-DIPOLE    0.0320        C  1 --  H  5          +49.1% ( 72.0%)    97.6%
 TRAVEL      0.1215        C  1 --  H  3           +0.8% (  9.5%)     0.0%
 RED. MASS   0.5026        C  1 --  H  2           +0.8% (  9.5%)     0.0%
 EFF. MASS   1.4730

 VIBRATION    9    9A       ATOM PAIR        ENERGY CONTRIBUTION    RADIAL
 FREQ.      3214.81        C  1 --  H  4          +25.0% ( 47.1%)   100.0%
 T-DIPOLE    0.0000        C  1 --  H  5          +25.0% ( 47.1%)   100.0%
 TRAVEL      0.1443        C  1 --  H  3          +25.0% ( 47.1%)   100.0%
 RED. MASS   0.2520        C  1 --  H  2          +25.0% ( 47.1%)   100.0%
 EFF. MASS   1.0079

           FORCE CONSTANT IN INTERNAL COORDINATES (Millidynes/A)

    ATOM   CHEMICAL  BOND LENGTH      BOND ANGLE     TWIST ANGLE
   NUMBER   SYMBOL  FORCE CONSTANT  FORCE CONSTANT  FORCE CONSTANT

      1       C        0.000000        0.000000        0.000000
      2       H        5.451689        0.000000        0.000000
      3       H        5.451689        0.811571        0.000000
      4       H        5.451689        0.811571        0.721397
      5       H        5.451689        0.811571        0.721397

          JOB ENDED NORMALLY



 TOTAL JOB TIME:             0.40 SECONDS

 == MOPAC DONE ==
